### ISSUE TITLE
Spanish style guide

### ISSUE THREAD

**Issue by dvidsilva** [angular/angular-es#38]
Creating this issue as a placeholder to talk about different Spanish style options. If anyone has concerns about words or style, let's talk about them before the translation grows too big.

Ideally we all use a similar, formal, style of Spanish, so that the documentation and content feels consistent.

**Comment by baruchvlz** [comment]
I agree. May I suggest, in order to create a proper style guide we might want to translate a few pages or partials and from those changes create the style guild and go on into translating bigger part of the website like the docs.

**Comment by dvidsilva** [comment]
I'm almost done with the 5 min quickstart and we can use it as reference.

**Comment by hmagrini** [comment]
I think that the first issue that comes to mind is: We need to consider that there are several definitions which are very specific and we might not want to translate. For example, lets consider `pull request`, the translation (literally is: `solicitud de extracción`) in Spanish really doesn't make any sense, because the meaning of a PR is tied to the context of Git, which is completely in English, and by translating such definitions, I feel that they loose that relationship. Also in normal conversation, even when speaking in Spanish, this definitions aren't translated, you say: "Podes revisar mi PR?" o "El PR para arreglar los problemas con el registro esta listo".

Also we need to understand the target audience of the translated docs. The audience is Spanish speaking developers, we want them to understand what they are reading, and learn what they don't know. If somebody doesn't know what a PR is, that person won't be able to find what `solicitud de extracción` means, cause in Spanish this is not used in any context that is remotely related to Git.

TL;DR: We should consider that not every word or definition will have a proper translation in Spanish, and that using the English version will be ok.

**Comment by Foxandxss** [comment]
Yes, you are 100% right. I will compile a list of words that we wouldn't translate.

As a coordinator between all translations, I will see which words can be translated in what languages.

Components and directives are some easy to translate in spanish, but stuff like data-binding is not.

**Comment by dvidsilva** [comment]
<img width="1151" alt="screen shot 2016-09-10 at 10 30 27 pm" src="https://cloud.githubusercontent.com/assets/1588753/18415426/c2f647c6-77a6-11e6-91b8-fa2ad95b6cdd.png">
Place holder para poder usar esta imagen en el readme.md
